src/api/app/policies/bs_request_policy.rb

```diff
@@ -13,4 +13,11 @@ def handle_request?
     is_author = record.creator == user.login
     record.state.in?([:new, :review, :declined]) && (is_target_maintainer || is_author)
   end
+
+  def add_reviews?
+    is_target_maintainer = record.is_target_maintainer?(user)
+    is_author = record.creator == user.login
+    has_open_reviews = record.reviews.where(state: 'new').select { |review| review.matches_user?(user) }.present?
+    record.state.in?([:new, :review]) && (is_author || is_target_maintainer || has_open_reviews.present?)
```

Comment:
Due to [this comment](https://github.com/openSUSE/open-build-service/pull/14335#pullrequestreview-1429987687) and the change you made, I think `.present?` should be dropped from `has_open_reviews` here now

Reply:
True, overlooked this. Done :)